importer/tests/test_import.py

```diff
@@ -109,3 +115,106 @@ def test_import_task(self):
             Course.objects.count(),
             original_count + 1,
         )
+
+    @staticmethod
+    def test_import_course_from_path():
+        """
+        Validate parameters and returns fir course from path.
+        """
+        test_path = '/tmp/foo'
+        test_repo_id = 2
+        test_user_id = 42
+        with mock.patch('importer.api.import_course') as mock_import:
+            with mock.patch(
+                'importer.api.import_static_assets'
+            ) as mock_static:
+                with mock.patch('importer.api.XBundle') as mock_bundle:
+                    with mock.patch('importer.api.isdir') as mock_is_dir:
+                        mock_import.return_value = True
+                        mock_is_dir.return_value = True
+                        import_course_from_path(
+                            test_path, test_repo_id, test_user_id
+                        )
+                        mock_import.assert_called_with(
+                            mock_bundle(), test_repo_id, test_user_id
+                        )
+                        mock_static.assert_called_with(
+                            os.path.join(test_path, 'static'), True
+                        )
+
+    def test_import_static_assets(self):
+        """
+        Verify walking a folder of assets and verifying they get added
+        """
+        temp_dir_path = mkdtemp()
+        self.addCleanup(rmtree, temp_dir_path)
+        basename = 'blah.txt'
+        file_contents = 'hello\n'
+        with open(os.path.join(temp_dir_path, basename), 'w') as temp:
+            temp.write(file_contents)
+        # All setup, now import
+        import_static_assets(temp_dir_path, self.course)
+        assets = StaticAsset.objects.filter(course=self.course)
+        self.assertEqual(assets.count(), 1)
+        asset = assets[0]
+        dummy = mock.MagicMock()
+        dummy.course = self.course
+        self.assertEqual(
+            asset.asset.name,
+            static_asset_basepath(dummy, basename)
+        )
+        self.addCleanup(default_storage.delete, asset.asset)
+
```

Comment:
Can you add a test or assertions for invalid path or course `import_static_assets`?